deploy/modules/aws/tidb-cluster/values/default.yaml

```diff
@@ -14,6 +14,17 @@ tidb:
       service.beta.kubernetes.io/aws-load-balancer-type: nlb
       service.beta.kubernetes.io/aws-load-balancer-cross-zone-load-balancing-enabled: 'true'
   separateSlowLog: true
+  config: |
+    [log]
+    level = "info"
+    [performance]
+    tcp-keep-alive = true
+  podSecurityContext:
+    sysctls:
+    - name: net.ipv4.tcp_keepalive_time
+      value: "300"
+    - name: net.ipv4.tcp_keepalive_intvl
+      value: "300"
```

Comment:
send keepalive packet every 300s to survive the 350s fixed idle timeout of AWS NLB.

Reply:
`net.ipv4.tcp_keepalive_intvl` defaults to 75 seconds, do you think it's necessary to increase it?

Reply:
I have no preference actually, just want to make sure the heartbeat packet interval is less 350 despite the information from kernel compiling time.

Reply:
I prefer not to change it. To prevent the connection from being closed by the load balancer which has shorter timeout, setting `net.ipv4.tcp_keepalive_time` is enough. `net.ipv4.tcp_keepalive_intvl` determines when the unresponding connection will be aborted. Increase it will increase the time the connection is kept on the server-side.

Reply:
Good point! But `net.ipv4.tcp_keepalive_intvl` determines the interval of subsequent probes, so it should be less than 350s too. I would like to set it to 75s explicitly (as the well-known defaults), how do you think?

Reply:
it's ok